manifests/m/Microsoft/VisualStudioCode/Insiders/1.58.0/Microsoft.VisualStudioCode.Insiders.locale.en-US.yaml

```diff
@@ -1,5 +1,5 @@
 PackageIdentifier: Microsoft.VisualStudioCode.Insiders
-PackageName: Microsoft Visual Studio Code Insiders
+PackageName: Visual Studio Code Insiders
```

Comment:
```suggestion
PackageName: Microsoft Visual Studio Code Insiders
```
![image](https://user-images.githubusercontent.com/15158490/122939732-2b3cbd00-d374-11eb-95e9-0ed9e525f0f7.png)

Reply:
@OfficialEsco, all packages that have been published through the winget manifest publishing pipeline are using `Visual Studio Code` and `Visual Studio Code Insiders`. That is also consistent with the package name that is being used for the Visual Studio manifests.